src/function/numerical/derivativeNum.js

```diff
@@ -0,0 +1,33 @@
+import { factory } from '../../utils/factory.js'
+
+const name = 'derivativeNum'
+const dependencies = ['typed', 'parse']
+
+export const createDerivativeNum = factory(name, dependencies, ({ typed, parse }) => {
+  return typed(name, {
+    /**
+     * Compute the derivate numerically of a function.
+     * The function considers the definition (f(x+dx)-f(x))/dx.
+     * It gets the function whose derivative we want to calculate as a string
+     * and it is parsed and evaluated by the library.
+     *
+     * Syntax:
+     *
+     * math.derivativeNum(string, number)
+     *
+     * Example:
+     *
+     * math.derivativeNum('cos(x)', 5)  // returns
```

Comment:
Can you add the actual result to the comment? (maybe pick these nice examples from your unit test)